### DISCUSSION TITLE
Localhost & Online Installer

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#110] [Q&A]
Saya coba installer Mecha kok tidak bisa ya mas? Itu baik yang online maupun offline.

Ini yang online, mas:

[Mecha CMS Test](hxxp://asdf.asdf.com/start.php)

**Comment by taufik-nurrohman** [comment]
Oh iya mba soalnya default _branch_ untuk Mecha sekarang semuanya sudah Saya ganti ke `main`, bukan `master`. Ntar tak dorong _patch_ baru ke [@mecha-cms/start](https://github.com/mecha-cms/start)

**Comment by taufik-nurrohman** [comment]
Sementara ini masih pakai `main` semua karena untuk pengembangan. Kalau mau spesifik memasang ke versi stabil bisa diganti konstanta [ini](https://github.com/mecha-cms/start/blob/ade97c019da1491dee0e8973eb73ce6cc6c5060a/start.php#L24-L26) ke tag-tag tertentu. Misal:
``` .php
define('THE_MECHA_VERSION', 'v2.5.3');
define('THE_PANEL_VERSION', 'v2.6.0');
define('THE_USER_VERSION', 'v1.12.1');
```

**Comment by taufik-nurrohman** [comment]
Coba login pakai `admin`

hxxp://asdf.asdf.com/user

**Reply by ghost** [reply]
Kalau saya buka [admin](https://asdf.asdf.com/user) bisa, tapi kan sebelumnya saya belum buat nama akun dan katakunci, Mas.

**Reply by ghost** [reply]
Oya Mecha kok tiba-tiba terinstall ya? Saya akses berandanya bisa.

**Reply by taufik-nurrohman** [reply]
Itu Saya yang buat 😉 pakai _element inspector_ untuk ngganti nilai `vmain` jadi `main` saja.

Oya sepertinya hostingan yang dipakai nggak memperbolehkan fungsi PHP `unlink()`? Karena setelah instalasi selesai seharusnya berkas instalasi akan dihapus.

**Reply by taufik-nurrohman** [reply]
Masalahnya cuma di nilai `<input name="d" type="hidden">` saja mba.
